app/Dockerfile

```diff
@@ -25,6 +30,7 @@ RUN mkdir -p /mattermost/data \
 
 # Configure entrypoint and command
 COPY entrypoint.sh /
+RUN chmod +x /entrypoint.sh
```

Comment:
Useless entry in my opinion. We just need to set the correct permission for the file on the repository, there is no need to do it on Dockerfile I guess.